src/apps/LandingRegister/Footer.react.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+export default React.createClass({
+  displayName: 'Footer',
+
+  getStyles() {
+    return {
+      footerDiv: {
+        display: 'flex',
```

Comment:
you need to use Radium here, as `display: 'flex'` isn't supported by all browsers